app/db.js

```diff
@@ -1,6 +1,17 @@
 'use strict';
 
+const env = process.env.NODE_ENV || 'development';
 const mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost/fadpack');
 
-module.exports = mongoose;
+const name = `fadpack-${env}`;
```

Comment:
just curious, value of ${env} will append to the string?
If so, what is the difference in doing 'fadpck-'+env